src/main/proto/marathon.proto

```diff
@@ -141,6 +141,22 @@ message MarathonTask {
     repeated string local_volume_ids = 1;
     required State state = 2;
   }
+  enum MarathonTaskStatus {
```

Comment:
Thanks for the note. I agree that src/main/scala/mesosphere/marathon/core/task/state/MarathonTaskStatus.scala should be a good place and link doc in Proto.